README.md

```diff
@@ -54,10 +54,7 @@ in the format of IP or hostname. The third argument `cli_path` is the full path
 | Package | Description |
 | --- | --- |
 | [/whisk.system/github](./packages/github/README.md) | offers a convenient way to use the [GitHub APIs](https://developer.github.com/). |
-| [/whisk.system/slack](./packages/watson-translator/README.md) | offers a convenient way to use the [Slack APIs](https://api.slack.com/). |
-| [/whisk.system/watson-translator](./packages/watson-translator/README.md) | Package for text translation and language identification|
-| [/whisk.system/watson-speechToText](./packages/watson-speechToText/README.md) | Package to convert speech into text|
-| [/whisk.system/watson-textToSpeech](./packages/watson-textToSpeech/README.md) | Package to convert text into speech|
+| [/whisk.system/slack](./packages/slack/README.md) | offers a convenient way to use the [Slack APIs](https://api.slack.com/). |
 | [/whisk.system/weather](./packages/weather/README.md) | Services from the Weather Company Data for IBM Bluemix API|
```

Comment:
Take out weather also?

Reply:
Was postponing Weather for more discussion. Easy enough to do in a separate PR.  Perhaps that is an interesting api (like Slack or Github) that it still makes sense to bundle it here as opposed to in an IBM repo.

Reply:
k makes sense.

Reply:
Weather can be remove in this or another PR